### ISSUE TITLE
Create report visualizer

### ISSUE THREAD

**Issue by thermokarst** [qiime2/q2-dada2#79]
Related to #77  

Convert stdout read fraction report to a visualization.

**Comment by ebolyen** [comment]
[forum x-ref](https://forum.qiime2.org/t/add-option-to-print-all-summary-statistics-of-dada2-plugin/2900)

**Comment by nbokulich** [comment]
[forum xref](https://forum.qiime2.org/t/log-for-a-running-dada2/3338?u=nicholas_bokulich)

**Comment by ebolyen** [comment]
[forum x-ref](https://forum.qiime2.org/t/output-stats-for-dada2/3731)

**Comment by ebolyen** [comment]
[forum x-ref](https://forum.qiime2.org/t/dada2-filtering-stats/3735/4)

**Comment by benjjneb** [comment]
This information is straightforward to write out from the R side, as long as there is a defined file format to use. My assumption is that is the way to go, since it sounds like you want that information as an artifact that can be visualized with other Q2 functions?

**Comment by ebolyen** [comment]
Yup! I think our plan at this point is to write it out as something like `SampleData[DADA2Stats]` or similar. This would be an artifact viewable as metadata and could be used with `qiime metadata tabulate` or wherever else metadata can be used. The only real work will be mapping the filepaths to sample IDs, but that's very simple also.
